### DISCUSSION TITLE
Developers tips from multiple YouTubers

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#280] [Ideas] [answered]
YouTubers record 1-3 tips for developers and I will create a mash up video and include their YouTube links in the video and the description.

- No need to edit the video, I will do it as part of the mashup video
- I think minimum of 8 people required
- About 1min of usable content (so if longer because of pauses between tips that is fine, as I will edit them out)
- If I could have it for the morning of Monday 5th that would be great

People confirmed:
- Bryan Jenks
- Adrian (hungryturtlecode)
- Jon Wood
- codeSTACKr
- Dennis Ivy
- Ali Spittel
- DThompsonDev
- James Perkins
- Brian Douglas
- Real Tough Candy
- Claudio Bernasconi
- Niall Maher
- Praveen (praveenscience)
- Debbie O'Brien

Videos received:
- [Ali Spittel](https://www.youtube.com/alispitteldev)
- [Jon Wood](https://www.youtube.com/c/JonWood)
- [Real Tough Candy](https://www.youtube.com/realtoughcandy)
- [Dennis Ivy](https://www.youtube.com/c/dennisivy)
- [Bryan Jenks](https://www.youtube.com/c/BryanJenksTech)
- [Danny Thompson](https://www.youtube.com/DThompsonDev)
- [Niall Maher](https://www.youtube.com/c/CodúCommunity)
- [Jesse Hall](https://www.youtube.com/codeSTACKr/)
- [Claudio Bernasconi](https://www.youtube.com/claudiobernasconi)
- [Adrian](https://www.youtube.com/channel/UC7Vxnf06GP6w42Lg3TQLXSw)
- [James](https://www.youtube.com/channel/UC-zet8Eyyy-OUE_fmocp3Wg)
- [Debbie O'Brien](https://www.youtube.com/channel/UCrNvYFsT1L3WczE8AizDQ6g)
- [Brian Douglas](https://www.youtube.com/user/ilikerobot)
- [Praveen Science](http://youtube.com/praveenscience)

If you are interested leave a comment below 👇

**Comment by codeSTACKr** [comment]
Count me in 😄 

- A developers best friend: Google
- The best way to learn: Jump in and code
- Dealing with impostor syndrome

**Reply by eddiejaoude** [reply]
Awesome, thanks @codeSTACKr 👍

**Comment by aspittel** [comment]
I'm in!

I take:

- move past tutorials and build something
- get comfortable with failure
- you don't need to know everything

**Reply by eddiejaoude** [reply]
Awesome 👍 . Thank you @aspittel 💪

**Comment by TheDThompsonDev** [comment]
I will do it on:

- Three rules to become a developer fast ABL, ABB, CCC
- Coding is hard, you aren't bad at it, it is what coding is!
- Supporting another person's success will NEVER ruin yours.

sound good? I can change them if you don't like something.

**Reply by eddiejaoude** [reply]
Sounds awesome @dannythompson901 👍

**Comment by tallguyjenks** [comment]
my 3
- build projects you would use yourself to propel your learning
- try to work around / with those who are more experienced than you
- get a rubber duck 🦆

**Reply by eddiejaoude** [reply]
Awesome 👍

**Comment by jwood803** [comment]
I got two so far:
- Actually get some sleep
- Do a little each day to improve your skills

**Reply by eddiejaoude** [reply]
Great 👍

**Comment by adiman9** [comment]
Mine:
- Focus on the smallest units when learning - don't get overwhelmed by the bigger picture
- Be adventurous, not everything has to be good - you are building good skills not good applications in the early days
- Scratch your own itch. Learning for learning sake is awesome but easy to lose motivation. Learning because you REALLY want the tool / product drives you through motivation slumps.

**Reply by eddiejaoude** [reply]
Cool 👍

**Comment by perkinsjr** [comment]
Mine:

-  Go slow, Go at your own pace.
-  Join a community (Such as a discord or forum)
- Don't compare yourself to others.

**Reply by eddiejaoude** [reply]
Nice 👍

**Comment by bdougie** [comment]
I love that you are using discussion for this ;)

tips:
- be ok with not knowing something. 
- be ok with struggling (but timebox it to, i.e. 30-90mins max) 
- when you get unblocked, write out how you did that on a short blog post

**Reply by eddiejaoude** [reply]
Sounds great 👍

**Comment by claudiobernasconi** [comment]
I'll work on the exact wording, but those are my 3 tips:

- Choose technology you like
- Write technical blog posts
- Work on fun side projects

**Reply by eddiejaoude** [reply]
Yay 👍

**Comment by Kratika0907** [comment]
It would be great if we can have something about code review as developer we spent lot of time reading other people's code.
My two cents :
1. Never fixate on formatting and linter rules. Keep a open mind about it.
2. Always check for readability and composability.

**Reply by eddiejaoude** [reply]
Great idea, thank you @Kratika0907 👍

**Comment by NiallJoeMaher** [comment]
1. Software is a social sport, it's rare anything substantial is built by a single person,  by learning to communicate effectively and it will take you further than you imagine. 
2. Get a mentor as soon as possible, by having someone you can lean on when you feel stuck or lost you can get out of ruts much faster than having an internal battle.
3. Listen to your body, if you feel tired or frustrated go for a walk, sleep or do something to refresh. It's tough to solve problems when fatigue or frustration is coming in.

**Reply by eddiejaoude** [reply]
Thanks @NiallJoeMaher

**Comment by praveenscience** [comment]
I have got a few:

1. Always be passionate and have the hunger for getting or developing something new.
2. Keep yourself very hydrated. Drink lots of water and take care of your diet.
3. Even though this is not possible, try getting more sleep whenever you can.
4. Finally, share your knowledge to the world!!!

**Reply by eddiejaoude** [reply]
Cool 👍

**Reply by praveenscience** [reply]
@eddiejaoude You may add my name in the confirmed list! 😜

**Reply by eddiejaoude** [reply]
Done, thank you 👍 - it is too late for me, past my bedtime 💤

**Comment by debs-obrien** [comment]
write out your problem
take a break when you are stuck
believe in yourself

**Reply by eddiejaoude** [reply]
thank you @debs-obrien 👍

**Comment by mikeysan** [comment, marked as answer]
I'm feeling the imposter syndrome at the moment as I trudge along my :snake:  journey. :see_no_evil:  I just jumped in the say these are really fantastic tips so far. The community will really love these.
@eddiejaoude Get the video out and let's get sharing, matey :smiley_cat:

**Reply by eddiejaoude** [reply]
thanks @mikeysan 👍 - the video will probably come out next week, as I just received more today.

Imposter syndrome sucks, and I find the community really helps me push through it. Let me know how we can help 🤓 

PS. thank you for going through those issues and inviting people to the organisation 💪
